src/TgSharp.Core/FileSessionStore.cs

```diff
@@ -48,6 +48,14 @@ public Session Load (string sessionUserId)
             }
         }
 
+        public void Delete (Session session)
+        {
+            string sessionFileName = $"{session.SessionUserId}.dat";
+            var sessionPath = basePath == null ? sessionFileName :
+                Path.Combine (basePath.FullName, sessionFileName);
+            File.Delete(sessionFileName);
```

Comment:
check if the file exist first.

Reply:
Ok, I'll commit another

Reply:
@aarani Should I throw Exception when not exists?

Reply:
no, just ignore the command if it doesn't